src/libraries/sendtohelixhelp.proj

```diff
@@ -361,11 +361,11 @@
     <PropertyGroup Condition="'$(RuntimeFlavor)' == 'CoreCLR' and '$(BUILD_BUILDID)' != ''">
       <HelixPostCommands Condition="'$(TargetsWindows)' == 'true'">
         $(HelixPostCommands);
-        %HELIX_PYTHONPATH% %HELIX_CORRELATION_PAYLOAD%\gen-debug-dump-docs.py -buildid $(BUILD_BUILDID) -workitem %HELIX_WORKITEM_FRIENDLYNAME% -jobid %HELIX_CORRELATION_ID% -outdir %HELIX_WORKITEM_UPLOAD_ROOT% -templatedir %HELIX_CORRELATION_PAYLOAD% -dumpdir %HELIX_DUMP_FOLDER%
+        %HELIX_PYTHONPATH% %HELIX_CORRELATION_PAYLOAD%\gen-debug-dump-docs.py -buildid $(BUILD_BUILDID) -workitem %HELIX_WORKITEM_FRIENDLYNAME% -jobid %HELIX_CORRELATION_ID% -outdir %HELIX_WORKITEM_UPLOAD_ROOT% -templatedir %HELIX_CORRELATION_PAYLOAD% -dumpdir %HELIX_DUMP_FOLDER% -productver $(ProductVersion)
       </HelixPostCommands>
       <HelixPostCommands Condition="'$(TargetsWindows)' != 'true'">
         $(HelixPostCommands);
-        $HELIX_PYTHONPATH $HELIX_CORRELATION_PAYLOAD/gen-debug-dump-docs.py -buildid $(BUILD_BUILDID) -workitem $HELIX_WORKITEM_FRIENDLYNAME -jobid $HELIX_CORRELATION_ID -outdir $HELIX_WORKITEM_UPLOAD_ROOT -templatedir $HELIX_CORRELATION_PAYLOAD -dumpdir $HELIX_DUMP_FOLDER
+        $HELIX_PYTHONPATH $HELIX_CORRELATION_PAYLOAD/gen-debug-dump-docs.py -buildid $(BUILD_BUILDID) -workitem $HELIX_WORKITEM_FRIENDLYNAME -jobid $HELIX_CORRELATION_ID -outdir $HELIX_WORKITEM_UPLOAD_ROOT -templatedir $HELIX_CORRELATION_PAYLOAD -dumpdir $HELIX_DUMP_FOLDER -productver $(ProductVersion)
```

Comment:
Did you test that product version is indeed set on this context?

Reply:
I tested it with:

```sh
$ cd runtime/
$ ./dotnet.sh build src/libraries/sendtohelixhelp.proj /v:diag /p:BUILD_BUILDID=123 | grep gen-debug-dump-docs 

                           $HELIX_PYTHONPATH $HELIX_CORRELATION_PAYLOAD/gen-debug-dump-docs.py -buildid 123 -workitem $HELIX_WORKITEM_FRIENDLYNAME -jobid $HELIX_CORRELATION_ID -outdir $HELIX_WORKITEM_UPLOAD_ROOT -templatedir $HELIX_CORRELATION_PAYLOAD -dumpdir $HELIX_DUMP_FOLDER -productver 7.0.0
```
notice the `-productver 7.0.0` printed at the end of last line.

Not sure how realistic this testing is compared to when / how it will be invoked by `helix.yml`, via `sendtohelix.proj`.

Reply:
It is pretty realistic, we do the same thing. Thanks for validating.

Reply:
It is working as expected. From: https://helix.dot.net/api/2019-06-17/jobs/b78eeb50-d32e-4f8b-9372-368124f56d1c/workitems/System.Drawing.Common.Tests/console

> 2021-10-01 21:02:05,521: INFO: 140339995743552: packing_test_reporter(33): report_results: Packed 1185962 bytes
> \+ python /root/helix/work/correlation/gen-debug-dump-docs.py -buildid 1398591 -workitem System.Drawing.Common.Tests -jobid b78eeb50-d32e-4f8b-9372-368124f56d1c -outdir /root/helix/work/workitem/uploads -templatedir /root/helix/work/correlation -dumpdir /home/helixbot/dotnetbuild/dumps -productver 7.0.0
Did not find dumps, skipping dump docs generation.
> \+ exit 0
(skipped because coredump wasn't generated, which seems normal by looking at older builds)

Reply:
Nice! Thank you!!